apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/TraceHeaderContext.cs

```diff
@@ -28,6 +28,8 @@ namespace Google.Cloud.Diagnostics.Common
     /// </remarks>
     internal sealed class TraceHeaderContext
     {
+        private static readonly TraceIdFactory _traceIdFactory = TraceIdFactory.Create();
```

Comment:
I was thinking that instead of having `TraceIdFactory` as a separate class, we could have `NextTraceId` as a method in this class. But without having looked at `TraceIdFactory` for a while, I don't know whether that's just a silly idea. Definitely no need to change right now, anyway.